doc/user-manual/tools/compilers.lagda.rst

```diff
@@ -140,13 +140,7 @@ typically faster and less readable.
 The :option:`--js-minify` flag makes the generated JavaScript code
 smaller and less readable.
 
-Agda can currently generate JavaScript modules in several flavours:
-
-The :option:`--js-es6` generates ES6-style modules (supported natively by browsers and NodeJS since 2020)
-
-The :option:`--js-cjs` (default) generates CommonJS-style modules (supported natively by NodeJS)
-
-The :option:`--js-amd` generates AMD-style modules, which are supported (for in-browser usage with a wrapper like `require.js`).
+Agda can currently generate JavaScript modules in ES6, AMD, or CommonJS style.
```

Comment:
I actually liked this overview over the possible options, why delete it?

Reply:
Because 5cm further down the page, there is an almost identical list of options.

Would you like me to put the extra detail into that section?

Reply:
c40ccd4 puts the extra detail into the `Options` section